test/cypress/integration/Dashboards/Components/KPI/KPIComponent.cy.js

```diff
@@ -4,15 +4,20 @@ describe('KPI Component test', () => {
     });
 
     it('KPI Component should have a last value of the point when connector attached.', () => {
-        cy.get('.highcharts-dashboards-component-kpi-value')
-            .should('be.visible')
+        cy.get('#kpi  .highcharts-dashboards-component-kpi-value').as('kpiConnector');
+        cy.get('#kpi-2  .highcharts-dashboards-component-kpi-value').as('kpi-val');
+            cy.get('@kpiConnector').should('be.visible')
             .should('have.text', 1200);
+
+            cy.get('@kpi-val').should('be.visible')
```

Comment:
What is the purpose of aliases when it's run one time? Especially in other tests, you do not declare them in similar scenarios.